packages/SwingSet/src/kernel/deviceManager.js

```diff
@@ -6,6 +6,17 @@ import { insistCapData } from '../capdata';
 import { insistMessage } from '../message';
 import kdebug from './kdebug';
 
+/**
+ * Produce an object that will serve as the kernel's handle onto a device.
```

Comment:
I think that may be a good terminological refactoring, but maybe as a separate pass, since it ends up touching a number of different parts of the code.

Reply:
Added issue #458